components/organisms/PokemonCard/PokemonCard.tsx

```diff
@@ -23,7 +20,12 @@ const PokemonCard = ({ name, number, id, image, style }: PokemonCardProps) => {
         Number: {number} - {name}
       </h3>
       <div className={styles["image-container"]}>
-        <Image src={image} layout="fill" objectFit="contain" alt={name} />
+        <Image
+          src={image ?? ""}
```

Comment:
Some optional chaining, because apparently, our API doesn't guarantee the existense of either the image or the name. (Likely a bug on their end, but hey, seems like the codegen tool is doing it's job)

Reply:
How did the codegen tool inspire this?